source/scripting_v3/GTA/Entities/Vehicles/Vehicle.cs

```diff
@@ -1050,6 +1086,21 @@ public float ThrottlePower
 		/// </summary>
 		public float MaxTraction => Function.Call<float>(Hash.GET_VEHICLE_MAX_TRACTION, Handle);
 
+		/// <summary>
+		/// Reduces this <see cref="Vehicle"/>'s current grip level.
+		/// In game versions prior to v1.0.1604.0, levels 1-3 are identical.
+		/// </summary>
+		/// <param name="level">The level from 0-3 to reduce grip by, with 0 being no reduction and 3 being maximum reduction.</param>
+		public void ReduceGripLevel(int level)
```

Comment:
I disassembled both `SET_VEHICLE_REDUCE_GRIP` and `SET_VEHICLE_REDUCE_GRIP_LEVEL`, and come to the conclusion that it the method should be separated into 2 separate methods.

For `SET_VEHICLE_REDUCE_GRIP`, the method could be changed to property and named like `HasLowerFrictionTires`, `HasReducedTireGrips`, or `AreTireGripsReduced`? That native simply sets a bit on the `CAutomobile` instance. Don't forget to indicate that the property can be set to that can only be applied to automobiles, heli, and planes (VehicleType <= 0xA), which are CAutomobile or its subclasses, in the document.

The doc of `SET_VEHICLE_REDUCE_GRIP`:
```
//INFO: 
//PARAM NOTES: only works on automobiles
//PURPOSE: Forces the car to have lower friction tyres
NATIVE PROC SET_VEHICLE_REDUCE_GRIP(VEHICLE_INDEX VehicleIndex, BOOL bEnableReduceGrip) = "0x03e7ca3dc742fa72"
```

For `SET_VEHICLE_REDUCE_GRIP_LEVEL`, we could name the new method like `SetReduceGripLevel`. Actually the native eventually changes the float value for reduced grip on individual tire struct though. The default reduce grip value is 0.11 (0x3DE147AE) btw.

The doc of `SET_VEHICLE_REDUCE_GRIP_LEVEL`:
```
//INFO: 
//PARAM NOTES: only works on automobiles
//PURPOSE: Sets the grip level to use, from 0 to 3, when the above function SET_VEHICLE_REDUCE_GRIP is called
NATIVE PROC SET_VEHICLE_REDUCE_GRIP_LEVEL(VEHICLE_INDEX VehicleIndex, INT ReducedGripLevel) = "0xbc1cdf8c3ef5fa95"
```